docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -120,19 +122,18 @@ $ for CELL in $(echo $RENAMED_CELLS); do
-<5> Assign all {compute_service} nodes from the source cloud `default` cell into `COMPUTES_CELL<X>` and `COMPUTES_API_CELL<X>``, where `<X>` is the `DEFAULT_CELL_NAME` environment variable value. In this example, the `DEFAULT_CELL_NAME` environment variable value equals `cell3`.
-<6> For each cell, update the `<["standalone.localdomain"]="192.168.122.100">` value and the `COMPUTES_API_CELL` value with the names and IP addresses of the {compute_service} nodes that are connected to the `ctlplane` and `internalapi` networks. Do not specify a real FQDN defined for each network. Use the same host name for each of its connected networks. Provide the IP addresses and the names of the hosts on the remaining networks of the source cloud as needed. Or you can manually adjust the files that you generate in step 9 of this procedure.
-<7> Cells that do not contain Compute nodes are omitted from this template because no node sets are created for the cells.
+* `export COMPUTES_CELL1=` defines the `<["standalone.localdomain"]="x.x.x.x">` value and the `COMPUTES_CELL<X>` value for each cell with the names and IP addresses of the {compute_service} nodes that are connected to the `ctlplane` and `internalapi` networks. Do not specify a real FQDN defined for each network. Always use the same hostname for each connected network of a Compute node. Provide the IP addresses and the names of the hosts on the remaining networks of the source cloud as needed. Or you can manually adjust the files that you generate in step 9 of this procedure.
+* `<compute1>`, `<compute2>`, and `<compute3>` specifies the names of your {compute_service} nodes for each cell. Assign all {compute_service} nodes from the source cloud `cell1` cell into `COMPUTES_CELL1`, and so on.
+* `export COMPUTES_CELL<X>=(` specifies all {compute_service} nodes that you assign from the source cloud `default` cell into `COMPUTES_CELL<X>` and `COMPUTES_API_CELL<X>``, where `<X>` is the `DEFAULT_CELL_NAME` environment variable value. In this example, the `DEFAULT_CELL_NAME` environment variable value equals `cell3`.
+* `export COMPUTES_API_CELL1=(` defines the `<["standalone.localdomain"]="192.168.122.100">` value and the `COMPUTES_API_CELL` value for each cell. `["standalone.localdomain"]="192.168.122.100"` defines the custom DNS domain in the FQDN value of the nodes. This value is used in the data plane node set `spec.nodes.<NODE NAME>.hostName`. Update these values with the names and IP addresses of the {compute_service} nodes that are connected to the `ctlplane` and `internalapi` networks. Do not specify a real FQDN defined for each network. Use the same host name for each of its connected networks. Provide the IP addresses and the names of the hosts on the remaining networks of the source cloud as needed. Or you can manually adjust the files that you generate in step 9 of this procedure.
```

Comment:
I don't get this change compared to the original meaning. This needs to stay closer to what it had been. ``COMPUTES_API_CELL`` is not a thing, ``COMPUTES_API_CELL1``..``COMPUTES_API_CELL<X>`` is

Reply:
We shouldn't use COMPUTES_API_CELL becaues there is no such variable.